README.md

```diff
@@ -33,11 +36,6 @@ This plugin helps you to read [Omnivore](https://omnivore.app/) articles and hig
 
 ![fetching](./fetching.gif)
 
-## Roadmap
```

Comment:
Haha. I noticed you removed this part

Reply:
@sywhb yeah i was just thinking it wasn't needed for install instructions since thats visible in the app.

Reply:
I agree!